### ISSUE TITLE
Error in logs

### ISSUE THREAD

**Issue by aequasi** [rolopogo/ValheimMods#2]
Logs getting spammed with this one:

```sh
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
ExploreTogether.Plugin.SendPin (Minimap+PinData pin, System.String text) (at <60cd288ae43740f5974a475107cc5ea9>:0)
ExploreTogether.Patches.Minimap_Patch.Minimap_UpdateProfilePins (Minimap __instance, Minimap+PinData& ___m_deathPin) (at <60cd288ae43740f5974a475107cc5ea9>:0)
(wrapper dynamic-method) Minimap.DMD<Minimap::UpdateProfilePins>(Minimap)
Minimap.UpdateDynamicPins (System.Single dt) (at <8e1e443e7dcc46fb93a437d981d93443>:0)
(wrapper dynamic-method) Minimap.DMD<Minimap::Update>(Minimap)

```

**Comment by SirQuacker** [comment]
yep getting the exact same error

**Comment by rolopogo** [comment]
This was caused by trying to send a null ping in Minimap's update. Related to the death pin spam issue
Resolved in 1.1.0
